### ISSUE TITLE
eslint-plugin-codegen - incompatible with eslint-plugin-markdown

### ISSUE THREAD

**Issue by wSedlacek** [mmkal/ts#235]
When using the [eslint-plugin-markdown](https://github.com/eslint/eslint-plugin-markdown) processor all markdown linting seems to fall over.

**Comment by mmkal** [comment]
Yes I can imagine why. Both plugins define a processor for `.md` files. I am not sure how this can be resolved though, if eslint scopes processors to files rather than rules - I will see if `eslint-plugin-codegen` can use the same processor as the other library but the general problem of competing processors for file types will probably remain unless fixed by eslint.

**Comment by papb** [comment]
I am facing this issue as well. @mmkal is there any related open issue on eslint? I found https://github.com/eslint/eslint/issues/11035 but it didn't seem what we really need here. How can I help?

**Comment by mmkal** [comment]
@papb thanks for the offer! Maybe the easiest way would be to set up a minimal repro? I’d be happy to then contribute/collaborate on a fix. eslint’s usage of dependencies are super weird so probably the surest way to figure out the fix would be to use patch-package in a reproduction repo, then we can port the fix to here. The js output is unminified so it should be doable.
